test/src/main/scala/scala/test/junit/specs2/Specs2Tests.scala

```diff
@@ -0,0 +1,16 @@
+package scala.test.junit.specs2
+
+import org.specs2.mutable.SpecWithJUnit
+import scala.test.junit.support.JUnitCompileTimeDep
+
+class JunitSpecs2Test extends SpecWithJUnit {
+
+  "specs2 tests" should {
+    "run smoothly in bazel" >> {
+      JUnitCompileTimeDep.hello
+      success
+    }
```

Comment:
I thought in the beginning I'll need to add such a test but didn't need to since the tests I added pushed very small parts of the code. I'm really confident about the regression here. In any case I'll add one per your request.